models/feedItem.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require( 'mongoose' );
+
+// create feedItem schema
+var options = { discriminatorKey: 'type' };
+
+var feedItemSchema = new mongoose.Schema({
+    _id: {
+        $oid: String
+    },
+    title: String,
+    description: String,
+    index: Number
+}, options);
+
+// create model
+var FeedItem = mongoose.model('feeditems', feedItemSchema);
```

Comment:
😑 How many minutes of my life should I expect to lose to bugs related to capitalization?